lib/transformations/resolve/resolve.js

```diff
@@ -0,0 +1,46 @@
+// Press ctrl+space for code completion
+export default function transformer(file, api) {
```

Comment:
Totally agree with that, my main concern at the moment is being able to understand what you are doing, so I can actually do code reviews and pick up some work too. At the moment, without context and understanding the ideas behind it, what the code is gonna do, etc, I am not able to do any of both. Thanks for the pointer you shared below!